lib/electron-inject.js

```diff
@@ -1,19 +1,21 @@
 ;(function () {
   var ipc
+
   try {
     ipc = require('electron').ipcRenderer
   } catch (e) {
     ipc = require('ipc')
   }
+
   window.zuul_msg_bus = []
-  ipc.on('started', loop)
-  window.setTimeout(ipc.send.bind(ipc, 'started'))
+
   function loop () {
     var msgs = window.zuul_msg_bus.splice(0, window.zuul_msg_bus.length)
-    msgs.forEach(send)
+    msgs.forEach(function (msg) {
+      ipc.send('zuulmessage', msg)
+    })
     setTimeout(loop, 100)
   }
-  function send (msg) {
-    ipc.send('zuulmessage', msg)
-  }
+
+  loop()
```

Comment:
I decided to remove all ipc back and forth between rendering and main process. Just start looping directly and do `ipc.send('zuulmessage', msg)`.

Reply:
On a side note, if we are to remove this IIFE, doesn't need a unique function name so not to clash with other code?

Reply:
Which IIFE are you referring to?

Reply:
The one in `lib/electron-inject.js`

Reply:
You mean the one that wraps everything, at line 1? That's a common pattern and fine to keep; in this case the IIFE actually improves readability IMO :) And you're right, it also means there's no name clash.